tests/test_eof.py

```diff
@@ -0,0 +1,30 @@
+import pytest
+
+from dahuffman.huffmancodec import _EOF
+
+
+def test_eq():
+    assert _EOF == _EOF
+
+
+@pytest.fixture(params=(b'a', b'\0', b'_EOF'))
+def raw_value(request):
+    return request.param
+
+
+@pytest.fixture(params=(lambda b: b.decode('utf-8'), bytes, tuple))
+def of_type(request):
+    return request.param
+
+
+@pytest.fixture
+def to_compare(raw_value, of_type):
```

Comment:
I feel this construct of three fixtures is a bit convoluted and misses some useful things to test against.
I would just make a straight list of values to use, e.g.
```python
[
 b"\0",
 b"abc",
 "abc",
 0, 
 100,
 -1000,
  False,
  True,
  None,
  [],
  (),
  {},
  [-100, 0, 10],
  (-100, 10),
  _EndOfFileSymbol()
]
```